src/backend/distributed/executor/adaptive_executor.c

```diff
@@ -4423,7 +4423,7 @@ RebuildWaitEventSetFlags(WaitEventSet *waitEventSet, List *sessionList)
 /*
  * SetLocalForceMaxQueryParallelization simply a C interface for
```

Comment:
SetLocalForceMaxQueryParallelization is simply a ....

Reply:
This makes sense. I will also update similar errors in https://github.com/citusdata/citus/blob/b7efb51e6354621374dd2727a78bc7160d270b2c/src/backend/distributed/executor/adaptive_executor.c#L4424

https://github.com/citusdata/citus/blob/b7efb51e6354621374dd2727a78bc7160d270b2c/src/backend/distributed/executor/local_executor.c#L797

and update PR title and description